.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,6 +31,20 @@ For example:
 
 - [ ] Verify that no errors appear in the JS console
 
+### C+ PR Review Checklist ([full steps here](../PR_REVIEW_CHECKLIST.md))
+<!--
+This is a checklist for Contributor Plus (C+) reviewers. If the PR is internal, these do not need to be checked off, but are still a nice reminder for what to look for when reviewing PRs.
```

Comment:
```suggestion
This is a checklist for Contributor Plus (C+) reviewers. If you are not an Expensify employee or a Contributor+, please leave this section blank, it will be used by the PR reviewer. If you are an Expensify Employee or Contributor+, these do not need to be checked off, but are still a nice reminder for what to look for when reviewing PRs.
```

Reply:
Hmm I'm pretty sure this checklist is supposed to _only_ be mandatory for C+ and "optional" for internal reviewers, am I wrong?